packages/storybook-vue/stories/components/telekom-brand-header-navigation/TelekomBrandHeader.stories.mdx

```diff
@@ -339,6 +340,7 @@ Set the profile menu's `logged-in` property to `true` or `false` to switch betwe
       loginHelpLabel: "Help logging in",
       registerHeadline: "Need an account?",
       registerLabel: "Register now",
+      type: "standard",
```

Comment:
Remove this since the 'standard' is not a type. it fixes the build if you remove this